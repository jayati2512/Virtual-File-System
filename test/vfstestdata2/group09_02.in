createvfs /iiitb 1024
createvfs aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa 1024
createvfs
createvfs myvfs 2024
createvfs myvfs 3
mountvfs /myvfs
mountfvfs myvfs
makedir / home
mountvfs myvfs
makedir / home
makedir / home
makedir /home temp
deletedir
deletedir bin
movedir /bin home
addfile /home file1 /usr/include/wait.h
addfile /home file2 /usr/include/wait.h
listfile /home/file2 /tmp/listfile.txt
listfile /home/file1 /tmp/listfile.txt
removefile /home/file1
exportfile /home/file1 /tmp/exportfile.txt
exportfile /home /tmp/xportfile.txt
searchfile ho 
searchfile ho /tmp/searchfile.txt
unmountvfs myvfs
