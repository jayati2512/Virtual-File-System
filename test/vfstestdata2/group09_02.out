createvfs_FAILURE INVALID_CHARACTER_IN_NAME
createvfs_FAILURE VFS_LABEL_TOO_LARGE
createvfs_FAILURE VFS_INSUFFICIENT_ARGUMENTS
createvfs_FAILURE INVALID_SIZE
createvfs_SUCCESS
mountvfs_FAILURE CANNOT_READ_FROM_FILE
mountvfs_SUCCESS
makedir_SUCCESS
mountvfs_FAILURE VFS_ALREADY_MOUNTED
makedir_FAILURE DIRECTORY_ALREADY_EXISTS
makedir_FAILURE DIRECTORY_ALREADY_EXISTS
makedir_SUCCESS
deletedir_FAILURE VFS_INSUFFICIENT_ARGUMENTS
deletedir_FAILURE CANNOT_FIND_SPECIFIED_DIR
movedir_FAILURE CANNOT_FIND_SPECIFIED_DESTINATIONDIR
addfile_SUCCESS
addfile_FAILURE:FILE_SYSTEM_FULL
listfile_FAILURE SOURCE_FILE_PATH_NOT_FOUND
listfile_FAILURE NOT_A_TEXT_FILE
removefile_FAILURE CANNOT_FIND_SPECIFIED_FILE
exportfile_FAILURE CANNOT_CREATE_OUTPUTFILE
exportfile_FAILURE CANNOT_EXPORT_DIR
searchfile_FAILURE VFS_INSUFFICIENT_ARGUMENTS
searchfile_SUCCESS  1
unmountvfs_SUCCESSIgnoring invalid command
